Documentation/CodeSnippets/Examples/Http/MeowInformationRequester.rst.txt

```diff
@@ -40,11 +40,17 @@
             ];
 
             // Get a PSR-7-compliant response object
-            $response = $this->requestFactory->request(
-                self::API_URL,
-                'GET',
-                $additionalOptions,
-            );
+            try {
+                $response = $this->requestFactory->request(
+                    self::API_URL,
+                    'GET',
+                    $additionalOptions,
+                );
+            } catch (\GuzzleHttp\Exception\ClientException $exception) {
+                if ($exception->code !== 200) {
+                   // Do something to continue execution
+                }
+            }
```

Comment:
The `ClientException` will only be thrown for HTTP status `4**`. So `$exception->code !== 200` is always true.

The `ServerException` will only be thrown for HTTP status `5**`

So for documentation I would prefer using `RequestException` which has a weighter range of catching exceptions:

```suggestion
            } catch (\GuzzleHttp\Exception\RequestException $exception) {
                // If a response could built above the exception code will be set to
                // the HTTP response code.
                if ($exception->code === 404) {
                   // Do something or protocol HTTP status 404 to log
                }
            }
```